utils/report.c

```diff
@@ -920,12 +947,16 @@ FIELD_TIME_DIFF_FULL_PCT(REPORT_F_SELF_TIME, self, self.sum, self, "Self time (d
 FIELD_TIME_DIFF_FULL_PCT(REPORT_F_SELF_TIME_AVG, self-avg, self.avg, self_avg, "Self avg (diff)");
 FIELD_TIME_DIFF_FULL_PCT(REPORT_F_SELF_TIME_MIN, self-min, self.min, self_min, "Self min (diff)");
 FIELD_TIME_DIFF_FULL_PCT(REPORT_F_SELF_TIME_MAX, self-max, self.max, self_max, "Self min (diff)");
-FIELD_CALL_DIFF_FULL(REPORT_F_CALL, call, call, call_diff_full_percent, "Calls (diff)");
+FIELD_OTHER_DIFF_FULL(REPORT_F_CALL, call, call, call_diff_full_percent, "Calls (diff)");
+FIELD_OTHER_DIFF_FULL(REPORT_F_PARENT, parent, parent, parent_diff_full_percent, "Parent (diff)");
+FIELD_OTHER_DIFF_FULL(REPORT_F_CHILD, child, child, child_diff_full_percent, "Child (diff)");
+FIELD_OTHER_DIFF_FULL(REPORT_F_DEPTH_MIN, depth-min, depth.min, depth_min_diff_full_percent, "Depth min (diff)");
+FIELD_OTHER_DIFF_FULL(REPORT_F_DEPTH_MAX, depth-max, depth.max, depth_max_diff_full_percent, "Depth max (diff)");
 
 FIELD_TIME(REPORT_F_TASK_TOTAL_TIME, total, total.sum, task_total, "Total time");
 FIELD_TIME(REPORT_F_TASK_SELF_TIME, self, self.sum, task_self, "Self time");
+FIELD_OTHER(REPORT_F_TASK_NR_FUNC, func, call, task_nr_func, "Num funcs");
 FIELD_TID(REPORT_F_TASK_TID, tid, task_tid, "TID");
-FIELD_CALL(REPORT_F_TASK_NR_FUNC, func, call, task_nr_func, "Num funcs");
```

Comment:
Please do not change the order.  This matches to actual ordering in `field_task_table` and in the output.